scheduler/src/cook/mesos/api.clj

```diff
@@ -2242,7 +2244,12 @@
                  (timers/time!
                    (timers/timer ["cook-scheduler" "handler" "list-endpoint-duration"])
                    (let [job-uuids (list-jobs db false ctx)]
-                     (mapv (partial fetch-job-map db framework-id) job-uuids))))))
+                     (histograms/update!
+                       (histograms/histogram ["cook-scheduler" "handler" "list-endpoint-response-count"])
+                       (count job-uuids))
+                     (timers/time!
+                       (timers/timer ["cook-scheduler" "handler" "list-endpoint-fetch-job-map-duration"])
+                       (mapv (partial fetch-job-map db framework-id) job-uuids)))))))
```

Comment:
Should we add this for `/jobs` as well?

Reply:
Not sure exactly what you're referring to by 'this'?

Reply:
By "this" I mean the new metric(s) that you're introducing for `/list`